### PR TITLE
Changed arrays token map semantic.

### PR DESCRIPTION
**Pull request by genegr**
Modified the lookup function for the tokens map, so that each array entry in the dict is matched with the 'endpoint' parameter of the query URL.

### CODE REVIEW

**Review by chrroberts-pure — COMMENTED** [review]
I like this implementation better, LGTM. 

Note: I haven’t tested the release and will update when I’ve had to chance to run this release.

File: examples/config/docker/prometheus/prometheus.yml [hunk 1]
```diff
@@ -1,84 +1,51 @@
 global:
-  scrape_interval:     60s
-  scrape_timeout:      50s
+  scrape_interval: 30s
+  scrape_timeout: 10s
```
[the remark this answers is not in the document]
Comment (sdodsley) [reply examples/config/docker/prometheus/prometheus.yml:3]: +1

File: examples/config/k8s/operator/pure-fa-exporter-deployment.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ spec:
     spec:
       containers:
       - name: pure-fa-om-exporter
-        image: quay.io/purestorage/pure-fa-om-exporter/pure-fa-om-exporter:1.0.4
+        image: quay.io/purestorage/pure-fa-om-exporter/pure-fa-om-exporter
```
[the remark this answers is not in the document]
Comment (sdodsley) [reply examples/config/k8s/operator/pure-fa-exporter-deployment.yaml:25]: +1

File: examples/config/docker/prometheus/prometheus.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (genegr) [reply examples/config/docker/prometheus/prometheus.yml]: Changed to 60s

File: examples/config/k8s/operator/pure-fa-exporter-deployment.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (genegr) [reply examples/config/k8s/operator/pure-fa-exporter-deployment.yaml]: fixed

**Review by sdodsley — APPROVED** [review]

**Pull request merged**